### PR TITLE
add support for legacy_org_id when creating/fetching orgs

### PR DESCRIPTION
**Pull request by mrmauer**
[Ticket](https://propelauth.atlassian.net/browse/TIC-851)
[BE PR](https://github.com/PropelAuth/auth/pull/641)

### CODE REVIEW

**Review by itailevi98 — APPROVED** [review]
lgtm!

**Comment by mrmauer** [comment]
retested w/ unrelated bug fix

File: src/models/fetch_org_response.rs [hunk 1]
```diff
@@ -29,12 +29,14 @@ pub struct FetchOrgResponse {
     pub org_id: String,
     pub name: String,
     pub is_saml_configured: bool,
-    #[serde(skip_serializing_if = "OrgMetadata::is_empty")]
+    #[serde(default, skip_serializing_if = "OrgMetadata::is_empty")]
```
Comment (mrmauer) [inline src/models/fetch_org_response.rs:32]: this fixes previous serialization issues

**Review by itailevi98 — APPROVED** [review]
lgtm!

**Pull request merged**